openwisp_notifications/tests/test_notification_setting.py

```diff
@@ -396,3 +397,36 @@ def test_validate_global_notification_setting(self):
             with self.assertRaises(ValidationError):
                 global_setting.full_clean()
                 global_setting.save()
+
+    @mock_notification_types
+    def test_preferences_api_excludes_disabled_organizations(self):
```

Comment:
This test is effective but should be placed in `test_api.py` for consistency with the existing tests for this view, which are placed there.